plugins/kiali/tests/kialiPage.spec.ts

```diff
@@ -0,0 +1,303 @@
+import { expect, Page, test } from '@playwright/test';
+
+import { Common, kialiData } from './kialiHelper';
+
+const CONFIG = kialiData.config;
+const NAMESPACES = kialiData.namespaces;
+const STATUS = kialiData.status;
+const BOOKINFO_APPS = kialiData.apps.bookinfo;
+const BOOKINFO_SERVICES = kialiData.services.bookinfo;
+const BOOKINFO_WORKLOADS = kialiData.workloads.bookinfo;
+
+async function checkReportedItems(
+  type: string,
+  ns: string,
+  objects: any,
+  page: any,
+): Promise<void> {
+  const ns_card = await page.locator(`data-test=overview-card-${ns}`);
+  await page.click('[aria-label="Health for"]');
+  await page.click(`[data-value=${type}]`);
+
+  const icon = await ns_card.locator('data-test=overview-app-health');
+  await icon.hover();
+  let list = await page.locator('data-test=overview-status');
+
+  // Wait for the list to appear
+  await page.waitForSelector('data-test=overview-status');
+
+  let i = 0;
+  for (const object of Object.entries(objects)) {
+    if (i == 5) {
+      break;
+    }
+    await icon.hover({ force: true }).then(async () => {
+      await expect(list).toContainText(`${object[1].name}`);
+    });
+    i++;
+  }
+  let expected = type == 'app' ? 'application' : type;
+  await expect(
+    ns_card.locator(`data-test=overview-type-${type}`),
+  ).toContainText(`${Object.entries(objects).length} ${expected}s`);
+}
+
+function filterByIstioInjectionEnabled(jsonArray: any): any {
+  return jsonArray.filter(
+    json =>
+      (json.labels && json.labels['istio-injection'] === 'enabled') ||
+      (json.name && json.name === 'istio-system'),
+  );
+}
+const visibleNamespaces = filterByIstioInjectionEnabled(NAMESPACES);
+
+test.describe('Kiali page', () => {
+  let page: Page;
+  let common: Common;
+  test.describe('overview', () => {
+    test.beforeAll(async ({ browser }) => {
+      const context = await browser.newContext();
+      page = await context.newPage();
+      common = new Common(page);
+      await common.loginAsGuest();
+    });
+    test.beforeEach(async () => {
+      page.reload();
+    });
+
+    test('Home cluster is visible', async () => {
+      await expect(page.locator('data-test=home-cluster')).toHaveText(
+        CONFIG.clusters.Kubernetes.name,
+      );
+    });
+
+    test('Debug info can be opened', async () => {
+      await page.click('data-test=help-button');
+      await page.click('text=View Debug Info');
+      await expect(
+        page.locator('css=[aria-describedby="Debug information"]'),
+      ).toBeVisible();
+    });
+
+    test('About page can be opened', async () => {
```

Comment:
Now the about page can be opened with just one click, so this test would need to be redone (Some steps less)